ext/js/display/display-anki.js

```diff
@@ -1121,6 +1122,28 @@ export class DisplayAnki {
         viewNoteButton.hidden = disabled;
         viewNoteButton.dataset.noteIds = noteIds.join(' ');
 
+        const cardStates = [];
+        for (const item of noteInfos) {
+            if (item === null) { continue; }
+            for (const cardInfo of item.cardsInfo) {
+                cardStates.push(cardInfo.cardState);
+            }
+        }
+
+        // Apply priority-based data-icon
+        const highestState = this._getHighestPriorityCardState(cardStates);
+        const dataIcon = /** @type {HTMLElement} */ (
+            viewNoteButton.querySelector('.icon[data-icon^="view-note-"]')
+        );
```

Comment:
Why is this wrapped in extra parentheses and taking up three lines?

Reply:
I made it take up one line, the extra parenthese is the only way I get get jsdocs to work for me, without throwing an warning and claiming dataIcon is a different type then I am declaring it as

Reply:
I seen this done in other parts of the codebase for reference